src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -22,9 +22,22 @@ public sealed partial class TimeZoneInfo
         private const string ZoneTabFileName = "zone.tab";
         private const string TimeZoneEnvironmentVariable = "TZ";
         private const string TimeZoneDirectoryEnvironmentVariable = "TZDIR";
+
+#if !TARGET_BROWSER
```

Comment:
Why browser only? We use the same data on other platforms like tvOS

Reply:
Is there a list of targets that use the filtered ICU data?  Or a different flag I should use?

These can all be trimmed out, anywhere that the full ICU with time zone data is not available.